hack/verify-gofmt.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright 2018 The cap Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+find_files() {
+  find . -not \( \
+      \( \
+        -wholename '*/vendor/*' \
+      \) -prune \
+    \) -name '*.go'
```

Comment:
go tooling should ignore vendor automagically by now.

Reply:
It should yes, but if I explicity tell gofmt to run against a file in vendor/* it will. So.. have to keep the -prune or tell it what directories to look in. Meh. :-)